transcriptic/commands.py

```diff
@@ -1476,6 +1476,28 @@ def run_protocol(api, manifest, protocol, inputs, view=False, dye_test=False):
             return
 
 
+def validate_filter(filters, number_of_instructions):
+    invalid_filters = set()
+
+    for arg in filters:
+        if arg.isdigit():
+            idx = int(arg)
+            if 0 > idx or idx >= number_of_instructions:
+                invalid_filters.add(arg)
+        elif re.match(r"\d+-\d+", arg):
+            [s, e] = [int(v) for v in arg.split("-")]
+            if s > e or number_of_instructions <= e:
```

Comment:
You should also check here that s is > 0

Reply:
I thought about it, I think '\d+' should enforce that right? 
With not 0?